packages/node/examples/calling/.env.example

```diff
@@ -1,5 +1,5 @@
-SIGNALWIRE_API_HOSTNAME=example.signalwire.com
-SIGNALWIRE_API_PROJECT=YOUR-PROJECT
-SIGNALWIRE_API_TOKEN=YOUR-TOKEN
+SIGNALWIRE_SPACE_URL=example.signalwire.com
+SIGNALWIRE_PROJECT=YOUR-PROJECT
```

Comment:
@bryanrite these env variables are useful to quick test the client and avoid to commit project/token. As for https://github.com/signalwire/signalwire-php/pull/16#issuecomment-474533584 maybe just changing the name?